postcode_lookup/template_sorter.py

```diff
@@ -328,12 +378,21 @@ def __init__(
             self.api_response, "registration", None
         )
         for i, date in enumerate(self.api_response.dates):
+            postal_vote_dispatch_dates = None
+            if (
+                self.electoral_services
+                # we only hold postal votes dispatch data data for one
+                # election. TODO: remove/review after 2025-05-01
+                and date.date == "2025-05-01"
```

Comment:
The data we have is only applicable to elections happening on this specific polling day, so we want to ignore this data for all other dates and just use the generic text.

However, in order to test this locally you need to comment this line out because the mock fixtures simulate where we are in the campaign period by fiddling the date of polling day relative to the current date.
Also to test locally, you need to insert some data in the file for Stroud as all the test data fixtures use
https://github.com/DemocracyClub/dc_response_builder/blob/e22b43068b013e179a2f3746bf6f2e1b4c68fa4d/response_builder/v1/generated_responses/electoral_services.py#L14-L23
but this is not one of the councils who have provided data about postal vote dispatch dates